src/index.html

```diff
@@ -6,7 +6,400 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="container">
+    <header class="header container__header">
+      <div class="logo">
+        <a href="#">
+          <img
+            src="images/logo.png"
+            alt="MoyoLogo"
+            class="logo__image"
+          >
+        </a>
+      </div>
+
+      <nav>
+        <div class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+              Laptops &amp; Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main container__main">
```

Comment:
same situation with container__main

Reply:
class container__main was for positioning